.travis.yml

```diff
@@ -7,14 +7,15 @@ install:
   - npm i
   - npm run bootstrap
   - npm run build
+  - ./circuits/scripts/installZkutil.sh
 
 script:
-  - ./contracts/scripts/compileSol.sh
   - cd crypto && npm run test
   - cd ../domainobjs && npm run test
   - cd ../core && npm run test
   - cd ../circuits && npm run buildBatchUpdateStateTreeSnark && npm run buildQuadVoteTallySnark
   # - ./scripts/buildSnarksSmall.sh
   - ./scripts/runTestsInCircleCi.sh
-  - cd ../contracts && ./scripts/runTestsInCircleCi.sh
+  - cd ../contracts && ./scripts/compileSol.sh
+  - ./scripts/runTestsInCircleCi.sh
```

Comment:
I tried to run contracts tests locally, but they are failing. One of the errors I see is this:

```
ENOENT: no such file or directory, open '.../maci/circuits/build/1601562991582.publicSignals.json'
```

Reply:
I see - please try the following:

```
cd circuits
./scripts/buildBatchUpdateStateTreeSnark.sh && ./scripts/buildQuadVoteTallySnark.sh
```

Reply:
Ah I forgot to set zkutil binary path :sweat_smile: 

Created PR to better handle such cases -> #182